sample-client-app/sample-shared-library/src/main/resources/sharedlib.properties

```diff
@@ -0,0 +1,27 @@
+# It's used for categorizing translation by product on Singleton side to maintain translation.
```

Comment:
Looks like this file is not used.

Reply:
This file will be used if sample-shared-library is used as a standalone app so let's keep it there.